boards/shields/mcp2515/mcp2515.conf

```diff
@@ -5,9 +5,5 @@ CONFIG_CAN_INIT_PRIORITY=80
 CONFIG_CAN_1=y
 CONFIG_CAN_STM32=n
 CONFIG_CAN_MCP2515=y
-CONFIG_GPIO_LED_DEV="GPIOB"
-CONFIG_PIN_LED_1=3
-CONFIG_GPIO_BUTTON_DEV="GPIOB"
-CONFIG_PIN_USER_BUTTON=4
 CONFIG_LOG=y
-CONFIG_CAN_LOG_LEVEL=4
\ No newline at end of file
+CONFIG_CAN_LOG_LEVEL=4
```

Comment:
@karstenkoenig I think CONFIG_LOG=y and CONFIG_CAN_LOG_LEVEL=4 should be removed for a shield/breakout board?  Also I think it's missing CONFIG_GPIO=y and CONFIG_SPI=y?

Reply:
Removed CONFIG_LOG stuff

CONFIG_SPI is there, CONFIG_GPIO I don't know, I'd assume that is pulled in elsewhere to be honest. Is it common to add extra?